docs/DeveloperGuide.md

```diff
@@ -81,6 +81,16 @@ The list medication command extends from Command parent class and contains the f
 then automatically checks off the relevant medication in the DailyMedication list, without user specifying 
 `(-m/-a/-e)` **after** `take -l [index]`.
 
+### View Medication Command
+The view medication command extends from Command parent class and contains the following:
+
+Given below is an example of and how the view command behaves at each step 
+with the help of a the sequence diagram with a usage scenario.
```

Comment:
Type "of a the "

Reply:
"with a usage scenario" sounds odd

Reply:
Yes, you are right. Good catch. It is resolved in  [a7a0246](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/242/commits/a7a02463300592ac68a0f52c248142e0473ce1ca), but still in progress.